### ISSUE TITLE
cc_matrix not downloading

### ISSUE THREAD

**Issue by Balidajr14** [facebookresearch/LASER#158]
Hi! I'm trying to download the CCMatrix using the script `dl_cc_matrix.py` but it I get this error

`requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://dl.fbaipublicfiles.com/laser/CCMatrix/v1.0/2018-34_tmp_0002.tsv.gz`

Any idea to fix it? 

Anyway, I'm interested specifically on English-Japanesse data, is there anyway to only download this language pair? Thank you very much!

David

**Comment by shenfe** [comment]
Same error encountered.

```
...
2020-10-31 19:37 INFO 3572200:root - Starting download of https://dl.fbaipublicfiles.com/laser/CCMatrix/v1.0/2018-34_tmp_0002.tsv.gz
Traceback (most recent call last):
  File "./dl_cc_matrix.py", line 147, in <module>
    dl(**vars(args))
  File "./dl_cc_matrix.py", line 77, in dl
    dl_file(metadata_dir, file, outdir)
  File "./dl_cc_matrix.py", line 95, in dl_file
    for i, line in enumerate(open_remote_file(metadata)):
  File ".../python3.7/site-packages/cc_net/jsonql.py", line 1124, in open_remote_file
    raw_bytes = request_get_content(url)
  File ".../python3.7/site-packages/cc_net/jsonql.py", line 1101, in request_get_content
    raise e
  File ".../python3.7/site-packages/cc_net/jsonql.py", line 1095, in request_get_content
    r.raise_for_status()
  File ".../python3.7/site-packages/requests/models.py", line 941, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://dl.fbaipublicfiles.com/laser/CCMatrix/v1.0/2018-34_tmp_0002.tsv.gz
```

Waiting for a fix.

**Comment by szhitansky** [comment]
I have the same 403 error.Probably list was removed or moved. Any information will be helpful, thanks!

**Comment by thpun** [comment]
list.txt is back now, but other files are not. and looks like list.txt is updated.

**Comment by gwenzek** [comment]
> https://dl.fbaipublicfiles.com/laser/CCMatrix/v1.0/2018-34_tmp_0002.tsv.gz

This was a wrong file that I uploaded by mistake I add to take it down. Sorry for the confusion.
We have a beta version of the data ready. 
We are still missing some pairs including the ones with English, Japanese and Chinese.
@hoschwenk  is working on that.

You can try downloading them by using 835c0708a285a667efd7f8316fda67877a2f4283 and

```sh 
pip install cc_net func_argparse
python tasks/CCMatrix/dl_cc_matrix.py dl --help
python tasks/CCMatrix/dl_cc_matrix.py dl -v v1.0.beta
```
Downloading is using 8 processes in parallel, you can increase that if your network isn't maxed out.
Once this is over you'll need `python tasks/CCMatrix/dl_cc_matrix.py finalize -v v1.0.beta` to extract the bitext.

Closing since this is a duplicate of #156
